exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,9 @@
 // Declare your function here
+function createLongGreeting(name, age) {
+    return "Hello, my name is" + " " + name + " " + "and I'm" +" " + age + " " + "years old";
```

Comment:
You can include the empty strings if they are next to strings. This would transform 
"Hello, my name is" + " " + name + " " + "and I'm" +" " + age + " " + "years old" 
into
"Hello, my name is " + name +  " and I'm " + age + " years old"